### PR TITLE
Extractcode libarchive system provided

### PR DESCRIPTION
**Pull request by aj4ayushjain**
This is the  plugins to be clubbed with the distro package which provides libarchive system package installed shared libraries.

### CODE REVIEW

File: plugins-builtin/extractcode-libarchive-system-provided/MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,9 @@
+graft src
+
+include setup.py
+include setup.cfg
+include .gitignore
+include README.md
+include MANIFEST.in
```
Comment (pombredanne) [inline plugins-builtin/extractcode-libarchive-system-provided/MANIFEST.in]: This is missing LICENSE.txt

File: plugins-builtin/extractcode-libarchive-system-provided/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+A scanCode debian package clubbed plugin to provide pre-built binary libraries,utilities and their locations.
```
Comment (pombredanne) [inline plugins-builtin/extractcode-libarchive-system-provided/README.md]: scanCode -> ScanCode
Also is this specific to Debian?

File: plugins-builtin/extractcode-libarchive-system_provided/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,143 @@
+This is the primary license for this library.
```
Comment (pombredanne) [inline plugins-builtin/extractcode-libarchive-system_provided/LICENSE.txt]: Is this file redundant with  plugins-builtin/extractcode-libarchive-system-provided/LICENSE.txt  https://github.com/nexB/scancode-toolkit/pull/1706/files#diff-22dc5702b8752ccfa9265468d79e3032 ?

File: plugins-builtin/extractcode-libarchive-system-provided/README.md (hunk 2 above)
Comment (aj4ayushjain) [reply plugins-builtin/extractcode-libarchive-system-provided/README.md]: Ack! Correctly pointed out these plugins are to be used with fedora package or other distribution package as well if needed.

File: plugins-builtin/extractcode-libarchive-system_provided/LICENSE.txt (hunk 3 above)
Comment (aj4ayushjain) [reply plugins-builtin/extractcode-libarchive-system_provided/LICENSE.txt]: Yes!

**Pull request merged**